### ISSUE TITLE
transform into workflow for feature prioritisation for both 1) high and 2) low dimensional data

### ISSUE THREAD

**Issue by liezeltamon** [liezeltamon/ml-tabular#9]
Create as final project for UK functional genomics course and make use of feedback from instructors
Refer to this guide - https://github.com/evanpeikon/Build_A_Biomarker/tree/main, https://sequenceanddestroy.substack.com/p/issue-69-what-actually-makes-someone

Ideas:
-Sklearn pipeline for high-dimensional data that can do feature selection, then llightgbm workflow for testing predictive power, integrate optuna + mlflow (check if there are alternatives)
-SHAP
-upskill: 1) Pipeline for biomarker discovery (see description) 2) ML application in work to get feel, maybe propagating cytof labels (use shap to see whether important feature for a cell type match with known marker, add uncertainty) or predicting labels from different types/levels of modules to see which ones most informative but do pipeline with heavy regularization (add uncertainty) at least using the key groups and use method for high dimensional data, emphasising also in splitting effectively based on clinical metadata, exploring or learning techniques for interpretability, could use pretrained scrnaseq foundational models with
